src/components/surfitProfile/Navbar.jsx

```diff
@@ -0,0 +1,52 @@
+import logo from 'assets/icon_logo.svg';
+import profilePic from 'assets/mockup_profile.svg';
+import { React } from 'react';
+import styled from 'styled-components';
+import { theme } from 'styles/theme';
+
+const NavContainer = styled.ul`
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  list-style: none;
+  width: 34.3rem;
+  height: 3.2rem;
+  font-size: 1.4rem;
+  color: ${theme.colors.surfit_grey_3};
+  margin: 5.2rem 0 auto;
+`;
+
+const Img = styled.img`
+  width: 3.2rem;
+  height: 3.2rem;
+`;
+
+const Logo = styled(Img)`
+  border: 1.5px solid white;
```

Comment:
```suggestion
  border: 1.5px solid ${theme.colors.surfit_white};
```
우리는 themeProvider사용하고 있고 그 안에 white색상까지 디쟌쌤들이 지정해주셨기때문에 이렇게 바꾸면 좋을거 같슴다~

Reply:
헉 다른데선 이렇게 써놓고 여기만 일케했네 수정할게이 ~